src/napari_matplotlib/line.py

```diff
@@ -0,0 +1,246 @@
+from typing import List, Optional, Tuple
+
+import matplotlib.colors as mcolor
+import napari
+import numpy as np
+import pandas as pd
+from magicgui import magicgui
+from magicgui.widgets import ComboBox
+
+from .base import NapariMPLWidget
+from .util import Interval
+
+__all__ = ["Line2DBaseWidget", "MetadataLine2DWidget"]
+
+
+class Line2DBaseWidget(NapariMPLWidget):
+    # opacity value for the lines
+    _line_alpha = 0.5
+    _lines = []
+
+    def __init__(self, napari_viewer: napari.viewer.Viewer):
+        super().__init__(napari_viewer)
+
+        self.axes = self.canvas.figure.subplots()
```

Comment:
The easier and recommended (because it sets up the right theme for the axes) way to do this - I think this is new since this PR was opened.
```suggestion
        self.add_single_axes()
```